### ISSUE TITLE
Forward mode jvp has nan gradients

### ISSUE THREAD

**Issue by ryanwu4** [sinaatalay/jaxellip#4]
I noticed that in this implementation, the forward mode jacobian of `ellipk` and `ellipe` returns NaNs. 

```
#test jacfwd and jacrev
@pytest.mark.parametrize("m", [0.1, 0.5, 0.9])
def test_ellipk_jacfwd_jacrev_agree(m):
    m_jax = jnp.array(m)
    J = jax.jacfwd(ellipk)(m_jax)
    J_rev = jax.jacrev(ellipk)(m_jax)
    assert J == J_rev
```

To mitigate this, perhaps a fast solution would just be to explicitly define the jvps for those functions analytically?

**Comment by sinaatalay** [comment]
Thank you and sorry for the late response. I've just released [v0.1.0](https://github.com/sinaatalay/jaxellip/releases/tag/v0.1.0), and this issue has been fixed.
